public-api/src/main/java/org/pmiops/workbench/google/GoogleAnalyticsServiceImpl.java

```diff
@@ -47,17 +51,25 @@ public GoogleAnalyticsServiceImpl(Provider<WorkbenchConfig> configProvider) {
      * @return int value of the http response.
      * @exception IOException if the URL could not be posted to
      */
+    @Override
     public int trackEventToGoogleAnalytics(
             String category, String action, String label, String value) throws IOException {
+        try {
+            Preconditions.checkNotNull(category);
```

Comment:
nit: I'd either just let this propagate, or use a normal if statement if you want the custom exception